MAINTAINERS.md

```diff
@@ -0,0 +1,84 @@
+This file explains the build and packaging commands of azure-chef-extension.
+
+##Contributors
+* [Siddheshwar More](https://github.com/siddheshwar-more)
+* [Nimisha Sharad](https://github.com/NimishaS)
+* [Vasundhara Jagdale](https://github.com/Vasu1105)
+* [Aliasgar Batterywala](https://github.com/Aliasgar16)
+
+##Build and Packaging
+You can use rake tasks to build and publish the new builds to Azure subscription.
+
+**Note:** The arguments have fix order and recommended to specify all for readability and avoiding confusion.
+
+#####Build
+    rake build[:target_type, :extension_version, :confirmation_required]
+
+:target_type = [windows/ubuntu/centos] default is windows
+
+:extension_version = Chef extension version, say 11.6 [pattern major.minor governed by Azure team]

```

Comment:
We don't use version #'s like this anymore right? We should use current versioning scheme as an example to avoid confusion.